OfficeAddin/04 Building Add-ins for Microsoft Outlook/Demos/03 Task Pane Experiences/src/commands/commands.html

```diff
@@ -1,25 +1,22 @@
-<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. -->
-
 <!DOCTYPE html>
 <html>
 
 <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
 
-    <!-- Office JavaScript API -->
-    <script type="text/javascript" src="https://appsforoffice.microsoft.com/lib/1.1/hosted/office.js"></script>
+  <!-- Office JavaScript API -->
+  <script type="text/javascript" src="https://appsforoffice.microsoft.com/lib/1.1/hosted/office.js"></script>
 
-    <script type="text/javascript" src="../node_modules/jquery/dist/jquery.js"></script>
-    <script type="text/javascript" src="../node_modules/showdown/dist/showdown.min.js"></script>
-    <script type="text/javascript" src="../node_modules/urijs/src/URI.min.js"></script>
-    <script type="text/javascript" src="../src/helpers/addin-config.js"></script>
-    <script type="text/javascript" src="../src/helpers/gist-api.js"></script>
+  <script type="text/javascript" src="../node_modules/jquery/dist/jquery.js"></script>
+  <script type="text/javascript" src="../node_modules/showdown/dist/showdown.min.js"></script>
+  <script type="text/javascript" src="../node_modules/urijs/src/URI.min.js"></script>
+  <script type="text/javascript" src="../src/helpers/addin-config.js"></script>
+  <script type="text/javascript" src="../src/helpers/gist-api.js"></script>
 </head>
 
 <body>
-  <!-- NOTE: The body is empty on purpose. Since functions in commands.js are
-       invoked via a button, there is no UI to render. -->
+  <!-- NOTE: The body is empty on purpose. Since functions in commands.js are invoked via a button, there is no UI to render. -->
```

Comment:
```suggestion
  <!-- NOTE: The body is empty by design. Since functions in commands.js are invoked using a button, there is no UI to render. -->
```